client/src/components/global/Nav.tsx

```diff
@@ -47,6 +47,8 @@ export default function HourLoggerNav() {
                   )}
                 </NavDropdown>
 
+                <Nav.Link href="/users">Users</Nav.Link>
```

Comment:
@avalittle wasn't sure which kinds of users should be able to see this. Let me know and ill update

Reply:
just admin